v2/refs/types.go

```diff
@@ -166,6 +170,19 @@ func (s *Signature) SetSign(v []byte) {
 	}
 }
 
+func (s *SubnetID) SetID(id uint32) {
```

Comment:
```suggestion
func (s *SubnetID) SetValue(id uint32) {
```

Pretty strange to set ID of the ID. Same for getter.